front/app/index.html

```diff
@@ -6,6 +6,9 @@
     <title>Rust Metrics Dashboard</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="author" href="humans.txt" />
+    <link rel="x-report-hack" href="https://github.com/dikaiosune/rust-dashboard/issues" />
```

Comment:
I'm not sure whether this should actually be the preferred reporting mechanism. I'd actually hope for someone to email me if there was a serious vulnerability.